skimage/feature/texture.py

```diff
@@ -291,3 +293,130 @@ def local_binary_pattern(image, P, R, method='default'):
     image = np.ascontiguousarray(image, dtype=np.double)
     output = _local_binary_pattern(image, P, R, methods[method.lower()])
     return output
+
+def multiblock_local_binary_pattern(int_image, x, y, width, height):
```

Comment:
@vighneshbirodkar 
@stefanv 
So, I guess our main priority now is to get first prototype faster
and then when it works, we can speed it with Cython.

I suggest that for now I will focus on the implementing the evaluation phase.
I will also will make a lot of useful measurements on the way.

@vighneshbirodkar I will tell you the timing today.